Answers.md

```diff
@@ -1 +1,20 @@
 
+- Describe the biggest difference between .forEach & .map.
+    .forEach loops through data and doesnt return anyhing.
+    .map loops through data and sets it in a new array.
+
+- What is the difference between a function and a method?
+    A function is a block of code that contains data to be executed later.
+    A Method is a piece of code created to run on an object 
+
+- What is closure?
+    Closure is when you have a function within a function and the nested function calls a variable from the parent function.
+
+- Describe the four rules of the 'this' keyword.
+    1. Window Binding is when this is not set on anything it automatically sets to window/default.
+    2. Implicit Binding refers .this to data within its function to call data used locally.
+    3. New Binding is used when you construct an object from a function. It instiantiates a new object of that type.
+    4. Explicit Binding is when you call, apply or bind data it makes a link to another __proto__ object. 
+
+- Why do we need super() in an extended class?
+    The super keyword is used to call functions or methods from a parent class.
```

Comment:
Pretty close, it calls the parent class with `this` bound to the same `this` in the child class.